C/jets.c

```diff
@@ -21,6 +21,14 @@ bool verify(frameItem* dst, frameItem src, const txEnv* env) {
   return readBit(&src);
 }
 
+/* low_1 : ONE |- TWO */
+bool low_1(frameItem* dst, frameItem src, const txEnv* env) {
+  (void) env; /* env is unused. */
+  (void) src; /* src is unused. */
+  writeBit(dst, 0);
```

Comment:
I could do this.  Do you think it would be better?

Reply:
> I could do this. Do you think it would be better?

Advantage:
 -  It would unify this implementation with the other implementations:
Disadvantage:
 - It decreases consistency a bit because it doesn't work for all operations.

So I'm not convinced that it's better. Both approaches are good enough. For the sake of avoiding work, I suggest keeping it as it is, unless you are convinced that the unified approach is better.